src/Card/Card.js

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { cardAlignOptions } from "./Card.levers";
+import { FlexBox } from "../FlexBox/FlexBox";
+import { Heading } from "../Heading/Heading";
+
+/**
+ * Card Component
+ *
+ * HTML card element
+ */
+export const Card = ({
+  className,
+  children,
+  image,
+  align,
+  isMinimal,
+  ...props
+}) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+
+  let wrapperClasses =
+    "su-bg-white su-text-black su-border su-border-solid su-border-black-10 su-shadow";
+
+  if (isMinimal) {
+    wrapperClasses = "";
+  }
+
+  // wrapperClasses = dcnb(wrapperClasses, bodyPadding, bodyAlign, bodyPadding);
+
+  return (
+    <article
+      className={dcnb(
+        "basic-card su-max-w-600 su-basefont-23",
+        wrapperClasses,
+        className
+      )}
+    >
+      {children}
+    </article>
+  );
+};
+
+// Subcomponents
+// ---------------------------------------------------------------------------
+const Content = ({ children, className, image, align, isMinimal }) => {
```

Comment:
@sherakama @yvonnetangsu 🤠  I added these props from the `Card` wrapper so that the example(s) would work within Storybook but I wonder if there's a more elegant way to pass props from the `<Card>` to `<Card.Content>` or if props should be passed in this way at all.

Reply:
Props shouldn't be passed down into the sub-components here. Each component and sub-component should have its own set of props. If multiple components need to "talk" to each other or know about each other's state, you will need to create a context. 


HeadlessUI is TS but you can see that Group, Switch, Label, and Description all have their own props. 
See: https://github.com/tailwindlabs/headlessui/blob/main/packages/%40headlessui-react/src/components/switch/switch.tsx#L37